common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala

```diff
@@ -186,4 +186,25 @@ class CelebornConfSuite extends CelebornFunSuite {
 
     assert(conf.testAlternative == "rss")
   }
+
+  test("Test empty working dir") {
+    val conf = new CelebornConf()
+    conf.set("celeborn.storage.activeLevels", "HDFS")
+    assert(conf.workerBaseDirs.isEmpty)
+
+    conf.set("celeborn.storage.activeLevels", "SDD,HDD,HDFS")
+    assert(conf.workerBaseDirs.isEmpty)
+
+    conf.set("celeborn.storage.activeLevels", "SDD,HDD")
+    assert(!conf.workerBaseDirs.isEmpty)
+  }
+
+  test("Test commit file threads") {
+    val conf = new CelebornConf()
+    conf.set("celeborn.storage.activeLevels", "HDFS")
+    assert(256 == conf.workerCommitThreads)
```

Comment:
The right pattern for scalatest assert is `assert(actual === expected)`

```suggestion
    assert(conf.workerCommitThreads === 256)
```